libcontainer/criurpc/criurpc.proto

```diff
@@ -1,3 +1,5 @@
+syntax = "proto2";
```

Comment:
Yes, it is. I tried to change proto2 to proto3 and get this error:
[avagin@laptop criu]$ make
  PBCC     images/rpc.pb-c.c
rpc.proto:1:10: Unrecognized syntax identifier "proto3".  This parser only recognizes "proto2".